parameters_meta_schema.json

```diff
@@ -0,0 +1,155 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://nextflow.io",
+  "title": "Nextflow Schema Meta-schema",
+  "description": "Meta-schema to validate Nextflow parameter schema files",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "title": "schema",
+      "type": "string",
+      "minLength": 1
+    },
+    "$id": {
+      "title": "ID URI",
+      "type": "string",
+      "minLength": 1
+    },
+    "title": {
+      "title": "Title",
+      "type": "string",
+      "minLength": 1
+    },
+    "description": {
+      "title": "Description",
+      "type": "string",
+      "minLength": 1
+    },
+    "type": {
+      "title": "Top level type",
+      "type": "string",
+      "const": "object"
+    },
+    "definitions": {
+      "title": "Parameter groups",
+      "type": "object",
+      "patternProperties": {
+        "^.*$": {
+          "type": "object",
+          "required": [
+            "title",
+            "type",
+            "properties"
+          ],
+          "properties": {
+            "title": {
+              "type": "string",
+              "minLength": 1
+            },
+            "type": {
+              "const": "object"
+            },
+            "fa_icon": {
+              "type": "string",
+              "pattern": "^fa"
```

Comment:
how about adding a description with a link to fontaweseome?

Reply:
Description is a Nextflow schema thing, not a JSON schema thing (it's a non-standard field).

We already document it here: https://nextflow-io.github.io/nf-validation/nextflow_schema/nextflow_schema_specification/#fa_icon